pandas/tests/frame/test_analytics.py

```diff
@@ -2078,9 +2079,6 @@ def test_n_error(self, df_main_dtypes, nselect_method, columns):
         col = columns[1]
         error_msg = self.dtype_error_msg_template.format(
             column=col, method=nselect_method, dtype=df[col].dtype)
-        # escape some characters that may be in the repr
-        error_msg = (error_msg.replace('(', '\\(').replace(")", "\\)")
```

Comment:
Why was this removed?

Reply:
I removed it as part of trying to hunt down some DeprecationWarnings for unescaped brackets (which I believe are somehow coming from pytest), and the tests still ran through. It is IMO an unnecessary misdirection that makes the test harder to read, but I reverted it.